tests/json_infra/test_trie.py

```diff
@@ -0,0 +1,114 @@
+import importlib
+import json
+from typing import Any
+
+import pytest
+from ethereum_types.bytes import Bytes
+
+from ethereum.utils.hexadecimal import (
+    has_hex_prefix,
+    hex_to_bytes,
+    remove_hex_prefix,
+)
+
+from . import FORKS, TEST_FIXTURES
+
+FIXTURE_PATH = TEST_FIXTURES["ethereum_tests"]["fixture_path"]
+
+forks = FORKS.keys()
+
+
+def to_bytes(data: str) -> Bytes:
```

Comment:
```suggestion
def to_bytes(data: str | None) -> Bytes:
```

based on the function contents

Reply:
Updated